galasa-parent/dev.galasa.framework.api.users/src/test/java/dev/galasa/framework/api/users/routes/UsersRouteTest.java

```diff
@@ -79,119 +53,65 @@ public void testUsersGetRequestWithMissingOrNullParamReturnsBadRequest() throws
-    }
-
-    @Test
-    public void testUsersGetRequestWithJwtUsernameNullReturnsBadRequest() throws Exception {
-        // Given...
-        MockUsersServlet servlet = new MockUsersServlet();
-        MockEnvironment env = new MockEnvironment();
-        env.setenv(EnvironmentVariables.GALASA_USERNAME_CLAIMS, null);
-        servlet.setEnvironment(env);
-
-        String requestorLoginId = "me";
-        Map<String, String[]> queryParams = new HashMap<>();
-
-        queryParams.put("loginId", new String[] { requestorLoginId });
-
-        MockHttpServletRequest mockRequest = new MockHttpServletRequest(queryParams, null, headerMap);
-        MockHttpServletResponse servletResponse = new MockHttpServletResponse();
-        ServletOutputStream outStream = servletResponse.getOutputStream();
-
-        // When...
-        servlet.init();
-        servlet.doGet(mockRequest, servletResponse);
-
-        // Then...
-        // Expecting this json:
-        // {
-        // "error_code" : 5058,
-        // [""GAL5058E: Unable to retrieve a username from the given JWT. No JWT claims
-        // to retrieve a username from were provided.
-        // This could be because the Galasa Ecosystem is badly configured.
-        // Report the problem to your Galasa Ecosystem owner.""]
-        // }
-        assertThat(servletResponse.getStatus()).isEqualTo(400);
-        checkErrorStructure(outStream.toString(), 5058, "GAL5058E", "Unable to retrieve a username from the given JWT");
-    }
-
-    @Test
-    public void testUsersGetRequestWithJwtUsernameEmptyStringReturnsBadRequest() throws Exception {
-        // Given...
-        MockUsersServlet servlet = new MockUsersServlet();
-        MockEnvironment env = new MockEnvironment();
-        env.setenv(EnvironmentVariables.GALASA_USERNAME_CLAIMS, "");
-        servlet.setEnvironment(env);
-
-        String requestorLoginId = "me";
-        Map<String, String[]> queryParams = new HashMap<>();
-
-        queryParams.put("loginId", new String[] { requestorLoginId });
-
-        MockHttpServletRequest mockRequest = new MockHttpServletRequest(queryParams, null, headerMap);
-        MockHttpServletResponse servletResponse = new MockHttpServletResponse();
-        ServletOutputStream outStream = servletResponse.getOutputStream();
-
-        // When...
-        servlet.init();
-        servlet.doGet(mockRequest, servletResponse);
+        String gotBackPayload = outStream.toString();
+        String expectedPayload = "[\n" +
+                "  {\n" +
+                "    \"login-id\": \"admin\",\n" +
+                "    \"id\": \"docid\",\n" +
+                "    \"clients\": [\n" +
+                "      {\n" +
+                "        \"last-login\": \"2024-10-18T14:49:50.096329Z\",\n" +
+                "        \"client-name\": \"web-ui\"\n" +
+                "      }\n" +
+                "    ]\n" +
+                "  }\n" +
+                "]";
```

Comment:
Hardcoding JSON can be really brittle and tricky to maintain. Instead, you could just build up a JsonArray containing JsonObjects and then do a `gson.toJson()` on that JsonArray to get it as a string.

Same applies for the tests below.